installer/install-rust-analyzer.sh

```diff
@@ -19,7 +19,7 @@ linux)
 darwin)
   if [[ $arch == "x86_64" ]]; then
     platform="x86_64-apple-darwin"
-  elif [[ $arch == "aarch64" ]]; then
+  elif [[ $arch == "arm64" ]]; then
```

Comment:
```
  elif [[ $arch == "aarch64" -o $arch == "amd64" ]]; then
```
I'm not well for apple mac but is this better? or not?

Reply:
Thank you for your suggestion. I think it's better. I will update.